config/default/package.json

```diff
@@ -13,6 +13,10 @@
     "test-targets": "npm run eslint && TZ=Africa/Nairobi mocha --reporter progress test/targets/*.spec.js --timeout 10000",
     "test-contact-summary": "npm run eslint && TZ=Africa/Nairobi mocha --reporter progress test/contact-summary/*.spec.js --timeout 10000",
     "test-contact-summary-extras": "npm run eslint && TZ=Africa/Nairobi mocha --reporter progress test/contact-summary-extras/*.spec.js --timeout 10000",
-    "test-unit": "TZ=Africa/Nairobi mocha --recursive --reporter spec test --timeout 20000"
+    "test-unit": "TZ=Africa/Nairobi mocha --recursive --reporter spec test --timeout 20000",
+    "compile-app-settings": "cht compile-app-settings"
+  },
+  "devDependencies": {
+    "eslint-webpack-plugin": "^2.7.0"
```

Comment:
Does it work if we only add this as a dev dependency in the root package.json file and keep the config one clean?